docs/UserGuide.md

```diff
@@ -252,6 +252,16 @@ Examples:
 * `sort o/name` returns the contact list sorted in alphabetical order.
 * `sort o/date` returns the contact list sorted in chronological order.
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+If sort is entered after executing find, a sorted found list will be displayed.
+However, the full address book will also be sorted.
```

Comment:
Minor thing but maybe instead of "however", we can phrase it into "The sorted state will be saved, so the full address book will also be sorted."?